SECURITY.md

```diff
@@ -1,7 +1,7 @@
 # Security
 
 > **IMPORTANT**: If you find a security issue, you can contact our team directly at
-security@tendermint.com, or report it to our [bug bounty program](https://hackerone.com/tendermint) on HackerOne. *DO NOT* open a public issue on the repository.
```

Comment:
do we want to start using security@interchain.io?

Reply:
I would turn down the email usage and use only the hackerone. It's more manageable with access etc...

Reply:
Yes I agree with @robert-zaremba using the Hackerone channel only is better. security@tendermint.com is also used for various other purposes.